indico/modules/events/layout/templates/layout.html

```diff
@@ -22,4 +36,16 @@
     {% call form_footer(form) %}
         <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}" data-disabled-until-change>
     {% endcall %}
+    <script>
+        $(document).ready(function() {
+            $('#theme').nullableselector({nullvalue: ''})
+            .on('change', function() {
+                var preview_url = build_url({{ url_for('event_layout.css_preview', event) | tojson }}, {
+                    theme: $(this).val()
+                });
+                $(this).nextAll('.i-form-field-button').attr('href', preview_url);
```

Comment:
Can't we set a custom class on the button to make it clear which button you are selecting?